odoo/openupgrade/doc/source/migration_details.rst

```diff
@@ -38,20 +76,11 @@ uncovered modules.
    Remember that the OpenUpgrade version of the source code is only intended to
    perform the migration, not run the Odoo server.
 
-General Tips
-++++++++++++
-
-* If you migrate your database through multiple versions make sure you use separate directories. If you use one directory and just check out a new branch the remaining *.pyc* files will cause trouble.
-
-* When installing the openupgradelib make sure you check out the latest version from github to get the latest updates and fixes. Either by following the method from *migrate.py* or installing via pip like so::
-
-    $ pip install git+git://github.com/OCA/openupgradelib.git
-
 Configuration options
 +++++++++++++++++++++
 
 OpenUpgrade allows for the following configuration options. Add these options
-to a separate stanza in the server configuration file under a header
+to a separate line in the server configuration file under a header
```

Comment:
I think `a separate stanza` (a group of lines) as a description of one or more lines is better than `a separate line` as a description of one or more lines.